libethash-cuda/keccak.cuh

```diff
@@ -328,12 +331,19 @@ __device__ __forceinline__ void keccak_f1600_init(uint2* s)
 
 	/* iota: a[0,0] ^= round constant */
 	s[0] ^= vectorize(keccak_round_constants[23]);
+
+	for(int i=0; i<12; ++i)
```

Comment:
Ah, sorry for my misunderstanding. I have checked in my code, please help to review. Thank you.